static/js/popovers.js

```diff
@@ -280,6 +366,19 @@ exports.register_click_handlers = function () {
         popovers.toggle_actions_popover(this, rows.id(row));
     });
 
+    $("#main_div").on("click", ".reaction_button", function (e) {
+        var row = $(this).closest(".message_row");
+        e.stopPropagation();
+        popovers.toggle_reactions_popover(this, rows.id(row));
+    });
+
+
+    $("body").on("click", ".reaction_button", function (e) {
+        var msgid = $(e.currentTarget).data('msgid');
```

Comment:
We should rename this to data-message-id to match the style for our other data objects.